src/deploy.ts

```diff
@@ -30,7 +32,7 @@ namespace Deploy {
      * @param namespace Namespace from which to delete deployment
      */
     export async function deleteDeployment(appSelector: string, namespaceArg?: string): Promise<void> {
-        ghCore.info("🔍 Checking for old deployments and deleting if found...");
+        ghCore.info("🔎 Checking for old deployments and deleting if found...");
```

Comment:
haha, why did the magnifying glass flip around? :mag: :mag_right:

Reply:
Motivation https://github.com/redhat-actions/openshift-tools-installer/blob/main/src/index.ts#L91 :wink:
Flipped one looks better :smile: